js/index.js

```diff
@@ -1 +1,62 @@
 // Your code here
+
+//invokes the clearWIndow() function.
+//generates a random number between 1 and 6 and passes the result to the function drawDots(). 
+function randomNum(){
+    clearWindow();
+    var random = Math.floor(Math.random() * 6) + 1;
+    drawDots(random);
+}
+
+//this function has a switch statement which goes through the possible cases (1-6) and adds the css .dot class to
+//the class of the appropriate id which I have specified in the html code.
+function drawDots(diceNum){
+    switch(diceNum){
```

Comment:
This function is too big! A function should just do one small job and you combine a bunch of them together to perform big tasks.